### ISSUE TITLE
[Feedback] Week 11 of 2019: Monday, March 11 – Sunday, March 17, 2019

### ISSUE THREAD

**Issue by funilrys** [Ultimate-Hosts-Blacklist/dev-center#19]
Hello, World!

Please report here every infrastructure/build/test issues (suspected or confirmed) of this week.

**Week covered by this issue:** Monday, March 11 – Sunday, March 17, 2019

**Note:** Issue that implies big changes or not related to infrastructure or build/test are welcome as a new issue!

Cheers,
Nissar

**Comment by dnmTX** [comment]
@funilrys  is it possible more then one list to be filtered in the same repo? If it is i would make suggestion to @smed79  if he want's to squeeze 😏  all of his lists in the same repo. I guess it would be easier to keep track and fetch them.
If it's not,maybe this could be the next feature project? 🛠

**Comment by smed79** [comment]
I apologise for my absence, I'm really busy this last while.
I will create a repo for all my lists over the next couple of days.

Again, I apologize I was not able to be here.

**Comment by dnmTX** [comment]
@smed79 thanks for considering it and welcome back 👍

**Comment by dnmTX** [comment]
@funilrys  just noticed that some lists didn't get filtered for a long time(some are showing 14+ days):
https://github.com/Ultimate-Hosts-Blacklist/SMed79_uponit.com_adservers
https://github.com/Ultimate-Hosts-Blacklist/smed79_getadmiral.com
https://github.com/Ultimate-Hosts-Blacklist/Spotify-Ad-free
There might be others as well. Can you check please. Thank you.

@mitchellkrogza  we had discussion here while ago what to do with one of [@smed79's repos](https://github.com/Ultimate-Hosts-Blacklist/SMed79_assorted) which has only three domains listed.
Suggestion was to be removed as @smed79 confirmed that he will not maintain it but @funilrys said that he needs to ask you first.Now that you here we can finish that task i suppose.

**Comment by funilrys** [comment]
Hello there, apologies for my days fully out of OSS.

Checks in my today's todo list.

About @smed79's repos,it's up to @smed79 to tell me what to do now.

**Comment by funilrys** [comment]
@dnmTX Those big one which are failling are because of the generation of the extra files which take some times. I may have to reoptimize again the process :sob:

**Comment by funilrys** [comment]
https://github.com/Ultimate-Hosts-Blacklist/SMed79_uponit.com_adservers ==> All right no changes.
https://github.com/Ultimate-Hosts-Blacklist/smed79_getadmiral.com ==> All right too.
https://github.com/Ultimate-Hosts-Blacklist/Spotify-Ad-free ==> Will look at it, it is not normal.

**Comment by funilrys** [comment]
https://github.com/Ultimate-Hosts-Blacklist/Spotify-Ad-free  ==> Travis Cron probably has a bug as the next run was planed for October 2019 :sob:. Restarted and adjusted it.

**Comment by mitchellkrogza** [comment]
We should probably drop this list from @smed79 for only 3 domains, those 3 domains, if still important can just be added to Badd-Boyz, seems like a waste of resources to keep testing only 3 domains :thinking:

**Comment by dnmTX** [comment]
@mitchellkrogza last time i checked i do believe that they're already present in Badd-Boyz.
Now we waiting on @smed79 to (cough) reconfirm (cough) 😏

**Comment by funilrys** [comment]
@mitchellkrogza, @smed79 already planned to merge some of his lists. I'm waiting for him :wink:

**Comment by smed79** [comment]
@dnmTX 

> i would make suggestion to **smed79** if he want's to squeeze smirk all of his lists in the same repo

check https://github.com/smed79/blacklist

(_please accept my apology. I was slow because i am very busy ..._)

@mitchellkrogza 

> We should probably drop this list from **smed79** for only 3 domains, those 3 domains, if still important can just be added to Badd-Boyz

You can move `cliqz.com` domains from "[smed79_assorted](https://github.com/Ultimate-Hosts-Blacklist/smed79_assorted)" to Badd-Boyz repo.

**Comment by mitchellkrogza** [comment]
Great @smed79 I will move cliqz.com into Badd-Boyz and let @funilrys handle bringing your new lists into Ultimate

**Comment by funilrys** [comment]
Note for self: Added to workflow. 

@mitchellkrogza Do not delete the repo !! You have to disable the CI first 🙂

**Comment by smed79** [comment]
@mitchellkrogza add
```
abtests.cliqz.com
analytics.cliqz.com
anolysis-gid.cliqz.com
anolysis-telemetry.cliqz.com
anolysis.privacy.cliqz.com
anti-tracking.cliqz.com
cliqz.com
collector-hpn.cliqz.com
collector-proxy-network.cliqz.com
collector.hpn.cliqz.com
hpn-collector.cliqz.com
reports.cliqz.com
repository.cliqz.com
stats.cliqz.com
tracker.cliqz.com
www.cliqz.com
```

**Comment by mitchellkrogza** [comment]
@funilrys I'll rather leave any deletion to you. 
@smed79 have added those above in latest commit.

**Comment by dnmTX** [comment]
@smed79 congrats on the new repo,already bookmarked.Good job 👍 💯

**Comment by funilrys** [comment]
@smed79 ==> Look at  https://github.com/mitchellkrogza/Badd-Boyz-Hosts issue 34 I need your opinion.
